robottelo/datafactory.py

```diff
@@ -66,6 +67,23 @@ def generate_strings_list(length=None, exclude_types=None, bug_id=None,
     return list(strings.values())
 
 
+def add_uppercase_char_into_string(text=None, length=10):
+    """Fix string to include a minimum of one uppercase character.
+    https://github.com/SatelliteQE/robottelo/issues/4742
+
+    :param string text : String to include uppercase character.
+    """
+    if text is None:
+        text = gen_string('alpha', length)
+    if text.lower() == text:
```

Comment:
Why do you care whether you string contain any upper case character at all? Especially when you say "fix string to include a **minimum** of one uppercase character."
Just add it